allure-robotframework/README.rst

```diff
@@ -25,7 +25,7 @@ Optional argument sets output directory. Example:
 
 .. code:: bash
 
-    $ robot --listener allure_robotframework;/set/your/path/here ./my_robot_test
```

Comment:
it is valid separator https://robotframework.org/robotframework/latest/RobotFrameworkUserGuide.html#id886